packages/sources/bitgo/src/index.ts

```diff
@@ -11,3 +11,4 @@ export const adapter = new PoRAdapter({
 })
 
 export const server = (): Promise<ServerInstance | undefined> => expose(adapter)
+export const walletParameters = wallet.inputParameters
```

Comment:
not sure if this is the right way to do this (also applies to the other EAs) but i also don't have any better suggestions 🤔

Reply:
You mean the idea or implementation ? I thought there should be a way to import the params without exporting them from index file. That ended up with lot of errors and didn't want to waste time there. 
There is really two ways, either this composite supports exact same params as underlying sources, or nothing and we forward the raw request body. 
Copy pasting source EA params is not good as any change will require an update to composite EA

Reply:
How about we don't do any definition, allow user to pass simply a json blob, this allows us to add and remove additional EAs in multi-address-list without code change

Reply:
the implementation, it's fine for now.